### PR TITLE
Chapter8

### PR DESCRIPTION
**Pull request by xtrxllx**
ready

### CODE REVIEW

**Comment by xtrxllx** [comment]
ready

**Review by saobv-2223 — COMMENTED** [review]
Resolve all the conflict files, please

**Comment by xtrxllx** [comment]
ready

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
-    </div>
-  </header>
+<header class="navbar navbar-fixed-top navbar-inverse">
+<div class="container">
+<%= link_to "sample app", "#", id: "logo" %>
+<nav>
+<ul class="nav navbar-nav navbar-right">
+<li><%= link_to t("global.home"), root_path %></li>
+<li><%= link_to t("global.help"), help_path %></li>
+<% if logged_in? %>
```
Comment (manhnt-0960) [inline app/views/layouts/_header.html.erb]: please check indent

**Comment by xtrxllx** [comment]
ready

**Comment by manhnt-0960** [comment]
merged

**Pull request merged**